### ISSUE TITLE
Proposed Enhancement: do not re parse already processed log lines on startup

### ISSUE THREAD

**Issue by johnfawcett** [fail2ban/fail2ban#2538]
Whenever I reboot the server where I have fail2ban installed, upon startup Fail2ban parses lines from the log files that have already been read. 

That can be time consuming and results in a lot of blocks being redone.

I think a better approach could be to save the state of fail2ban when it stops (including existing blocks and location in log files) and then on startup restore the state.

**Comment by sebres** [comment]
> upon startup Fail2ban parses lines from the log files that have already been read.

Yes, it does... but only the messages from interval `now - findtime`. And only starting from last position in log (unless database is disabled or fail2ban version is below 0.9).
If you don't need it, you can specify parameter `tail` in option `logpath` after filename. See related section in [documentation](https://manpages.debian.org/testing/fail2ban/jail.conf.5.en.html#JAIL_CONFIGURATION_FILE(S)_(jail.conf)).
